subprojects/core-api/src/main/java/org/gradle/StartParameter.java

```diff
@@ -850,4 +851,24 @@ public boolean isInteractive() {
     public void setInteractive(boolean interactive) {
         this.interactive = interactive;
     }
+
+    /**
+     * Whether caching for native tasks is enabled.
+     *
+     * @since 4.3
+     */
+    @Incubating
+    public void setNativeCachingEnabled(boolean nativeCachingEnabled) {
+        this.nativeCachingEnabled = nativeCachingEnabled;
+    }
+
+    /**
+     * Whether caching for native tasks is enabled.
+     *
+     * @since 4.3
+     */
+    @Incubating
+    public boolean isNativeCachingEnabled() {
+        return nativeCachingEnabled;
+    }
```

Comment:
It would be great if we had some way to mark this as to be removed soon. Maybe mark it as `@Deprecated` already?